### ISSUE TITLE
gpu request

### ISSUE THREAD

**Issue by moro0v0** [thu-coai/SafeUnlearning#2]
My environment is 2x3090-24g. Can I run the code?

**Comment by yangjunx21** [comment]
You could try setting batch size to 1 and using gradient accumulation to achieve the expected batch size. However, training a 7B model on 2×3090 (24GB) would be quite challenging.
